spectator-ext-ipc/src/main/java/com/netflix/spectator/ipc/http/HttpRequestBuilder.java

```diff
@@ -293,7 +293,11 @@ public HttpResponse send() throws IOException {
         if (attempt == numAttempts || !retryPolicy.shouldRetry(method, e)) {
           throw e;
         } else {
-          LOGGER.warn("attempt {} of {} failed: {} {}", attempt, numAttempts, method, uri);
+          if (attempt < numAttempts) {
+            LOGGER.info("attempt {} of {} failed: {} {}", attempt, numAttempts, method, uri);
+          } else {
```

Comment:
I think this `else` condition is unreachable. Or am I missing something?

Reply:
Yep, good catch. Fixed.